scripts/install/run.sh

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env bash
+set -e
+
+# Prints header
+function printh {
+    local input="$1"
+    local len=$((${#1}+2))
+    printf "\033[32m" # start green color
+    printf "\n+"
+    printf -- "-%.0s" $(seq 1 $len)
+    printf "+\n| $input |\n+"
+    printf -- "-%.0s" $(seq 1 $len)
+    printf "+\n\n"
+    printf "\033[0m" # end green color
+}
+
+function integrate_with_smartedit {
+  cmd_install "$@"
+
+  cp ${CLONE_DIR}/projects/storefrontapp/src/webApplicationInjector.js ${APP_DIR}/src
+
+  jq ".projects.${APP_NAME}.architect.build.options.assets += [\"src/webApplicationInjector.js\"]" ${APP_DIR}/angular.json > ${APP_DIR}/temp.json
+
+  mv ${APP_DIR}/temp.json ${APP_DIR}/angular.json
+}
+
+function delete_dir {
+    if [ -d "${1}" ]; then
+        echo "deleting ./${1}"
+        rm -rf ${1}
+    fi
+}
+
+function cmd_clean {
+    printh "Cleaning Spartacus installation workspace"
+
+    delete_dir $CLONE_DIR
+    delete_dir $INSTALLATION_DIR
+    delete_dir storage
+}
+
+function pre_install {
+    if [ -d $BASE_DIR ]; then
+        echo "Directory ${BASE_DIR} already exists, please remove it in order to start fresh installation."
+        exit 1
+    fi
+
+    npm config set @spartacus:registry https://registry.npmjs.org/
+
+    cmd_clean
+
+    yarn cache clean
+
+    npm i -g verdaccio
+    npm i -g serve
+    npm i -g pm2
+    npm i -g concurrently
+    npm i -g @angular/cli@${ANGULAR_CLI_VERSION}
+
+    mkdir -p ${INSTALLATION_DIR}
+}
+
+function clone_repo {
+    printh "Cloning Spartacus installation repo"
+
+    echo "Cloning from ${SPARTACUS_REPO_URL}"
+    echo "Branch: ${BRANCH}"
+
+    git clone -b ${BRANCH} ${SPARTACUS_REPO_URL} ${CLONE_DIR} --depth 1
+}
+
+function update_projects_versions {
+    projects=$@
+    for i in $projects
+        do
+            echo $i;
+            (cd "${CLONE_DIR}/projects/${i}" && pwd && sed -i -E 's/"version": "[^"]+/"version": "'"${SPARTACUS_VERSION}"'/g' package.json);
```

Comment:
Can we fix the case when someone passes `latest` or `next` here in local install? Then it won't work. Publish will fail with those 2. We could set it maybe for very high version `999.999.999`? What you think?

Reply:
What's the use case for passing `latest` or `next` in local install?